pom.xml

```diff
@@ -63,6 +57,67 @@
 
 	<profiles>
 		<profile>
+			<id>ossrh</id>
+			<distributionManagement>
```

Comment:
Cool. Thanks for the tip anyway. We'll try it like this and see what happens. I kinda need to use the staging plugin because we are configuring this to be deployed and staged/promoted from Hudson, rather than from the command line.